lib/tasks/push-to-space/creation.js

```diff
@@ -49,6 +49,9 @@ function createEntry (entry, target, skipContentModel, destinationEntries) {
 
 function updateDestinationWithSourceData (destinationEntity, sourceEntity) {
   const plainData = getPlainData(sourceEntity)
+  if (destinationEntity.default) {
+    delete destinationEntity.default
+  }
```

Comment:
I don't quite understand why this would suddenly pop up when we fixed the archiving logic.  
Maybe it was just that the test case there was wrong?

But also, it seems totally unrelated to locales, so I really have no idea what's going on here.

Reply:
Yeah totally agree. will spend a bit more time looking into it this afternoon

Reply:
If we need to start removing `default` keys, I'd like to add that as a separate PR

Reply:
Aren't we removing that already [here](https://github.com/contentful/contentful-import/blob/master/lib/transform/transformers.js#L54)?